app/components/user/profile_dropdown_component.html.erb

```diff
@@ -28,7 +28,7 @@
       <%= render Theme::SwitcherComponent.new(current_theme:) %>
     <% end %>
 
-    <%= link_to sign_out_path, class: 'text-gray-700 dark:text-gray-300 group flex items-center px-3 py-2 text-sm' do %>
+    <%= link_to sign_out_path, class: 'text-gray-700 dark:text-gray-300 group flex items-center px-3 py-2 text-sm', data: { turbo_method: :delete } do %>
```

Comment:
Would it be better to use `button_to` instead of link to to improve the semantics, as well? I know there are some considerations with how `button_to` wraps the button in a form, but could still be worthwhile